srcCpp03/RTIDDSImpl.cxx

```diff
@@ -205,6 +206,15 @@ bool RTIDDSImpl<T>::validate_input()
                 _PM->set<long>("batchSize", 0); // Disable Batching
             }
         }
+
+        if (_isFlatData) {
+            if (_PM->is_set("batchSize") && _PM->get<long>("batchSize") > 0) {
+                fprintf(stderr, "Batching cannot be used with FlatData.\n");
```

Comment:
We should be using `std::cerr`, but I see we use fprintf in the rest of the file.

Reply:
I have changed all `printf(stderr)` to `std::cerr`.